util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisService.java

```diff
@@ -33,7 +33,9 @@ private SystematicSensitivityAnalysisService() {
 
     public static SystematicSensitivityAnalysisResult runAnalysis(Network network,
                                                                   Crac crac,
-                                                                  ComputationManager computationManager) {
+                                                                  ComputationManager computationManager,
+                                                                  boolean dcMode,
+                                                                  boolean acToDcFallback) {
```

Comment:
The parameters are not used !!

Reply:
Sure, the aim of the dev was to import the parameters, not to implement the whole US...

Reply:
:package: